src/Features/Core/Portable/EditAndContinue/EmitSolutionUpdateResults.cs

```diff
@@ -74,6 +74,127 @@ internal ImmutableArray<ManagedHotReloadDiagnostic> GetAllDiagnostics()
 
             return builder.ToImmutableAndClear();
         }
+
```

Comment:
Not sure why this is shown as added. Did you rebase against the latest main?

Reply:
Not quite follow, what you mean here

Reply:
Looks like the code was copied from https://github.com/dotnet/roslyn/pull/75503/files#diff-95af090e3bc672c6352c9245c5b43e6d1561613fc81afee422e462dc9f17a49fR266.

Reply:
Yeah that's true, would be better to find out a way to clean it up

Reply:
Let me look into that